conda_smithy/schema.py

```diff
@@ -172,6 +172,11 @@ class AzureConfig(BaseModel):
         description="The name of the Azure Pipelines project",
     )
 
+    build_id: Optional[int] = Field(
+        default=None,
+        description="The Build ID for the specific feedstock",
```

Comment:
Since this will appear in our docs, can we add an explanation of what this does and why one might use it?